identity/authentication_test.go

```diff
@@ -149,13 +150,17 @@ func TestHandler_florenceToken(t *testing.T) {
 				So(common.Caller(ctx), ShouldEqual, userIdentifier)
 				So(common.User(ctx), ShouldEqual, userIdentifier)
 			})
+
+			Convey("And authResp.Body is closed", func() {
```

Comment:
minor point, but `authResp` is not meaningful here, either expand to English or possibly:
```suggestion
			Convey("And `body` is closed", func() {
```

similar copy-pasta below